Content.Server/Traitor/Uplink/UplinkComponent.cs

```diff
@@ -22,6 +23,8 @@ public sealed class UplinkComponent : Component
 
         [ViewVariables] public UplinkAccount? UplinkAccount;
 
+        [ViewVariables] public HashSet<JobPrototype>? JobWhitelist = null;
```

Comment:
Just use a string + typeserializer for it then you don't need jobprototype to be serializable. Also this doesn't have a datafield.